From a7659430f008c14d94534e0fd9330670a50aca79 Mon Sep 17 00:00:00 2001
From: Guy Nisan <gnisan@habana.ai>
Date: Mon, 29 Mar 2021 10:34:52 +0300
Subject: [PATCH] [SW-36796] i2c: add preboot device status

When HLX queries for device status it expects value 2 for preboot when
firmware is in appropriate stage.

Change-Id: I91af182d8dba55f68574edde797651c65b8170e2
Signed-off-by: Guy Nisan <gnisan@habana.ai>
---

diff --git a/preboot/module_i2c_slave.c b/preboot/module_i2c_slave.c
index cdcca79..35b90ad 100644
--- a/preboot/module_i2c_slave.c
+++ b/preboot/module_i2c_slave.c
@@ -40,7 +40,7 @@
 		ret = HL_DEVICE_STATUS_CMD_CODE_LEN;
 		break;
 	case HL_DEVICE_STATUS:
-		ret = HL_DEVICE_STATUS_UBOOT;
+		ret = HL_DEVICE_STATUS_PREBOOT;
 		break;
 	case HL_PCIE_VENDOR_ID_VAL_LOW:
 	case HL_PCIE_VENDOR_ID_VAL_HIGH:
